samples/snippets/helpers.py

```diff
@@ -5,7 +5,7 @@
 
 
 def get_name(out, key="name"):
-    pattern = re.compile(fr'{key}:\s*"([\-a-zA-Z0-9/]+)"')
+    pattern = re.compile(fr'{key}:\s*"([\_\-a-zA-Z0-9/]+)"')
```

Comment:
Hmm, since this procedure is used across many featurestore tests, is it ok that instead of changing existing function, I create a new function within `helpers.py`?

Reply:
I would wait to see the tests pre submit results, if no other tests are affected by this change, we don't need to create another helper function.